store/storeapp/views.py

```diff
@@ -1,3 +1,49 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
+from django.utils import timezone
+from store.storeapp.models import ClientModel, OrderModel
```

Comment:
```suggestion
from storeapp.models import ClientModel, OrderModel
```
В командах тоже неверные импорты

Reply:
В команде create_product вместо функции paragraphs вы почему-то используете paragraph, от чего происходит ошибка

Reply:
Команда create_order тоже написано неверно. Вот как должно быть
```python
class Command(BaseCommand):

    def handle(self, *args, **kwargs):
        order = OrderModel(
            client=choice(ClientModel.objects.all()),
            total_price=0,
        )
        order.save()
        for _ in range(10):
            product = choice(ProductModel.objects.all())
            order.products.add(product)
            order.total_price += product.price

        print('Add new order')
```